### PR TITLE
Initial pass at updating script

### PR DESCRIPTION
**Pull request by cowanwalls**
# Description

Script changes following QA feedback

## Changes

- `templates/routing-informs/part1.sh`
    - Updated to `#!/bin/bash` from `#!/bin/sh`
    - Added a "letters only" indicator to users entering their initials
    - Force `ID` variable from initials to lowercase using `tr` command
    - Adjusted spacing of messages to not exceed 80 characters per line
- `templates/routing-informs/part2.sh`
    - Updated to `#!/bin/bash` from `#!/bin/sh`
    - Updated format of commands printed to user to be consistent with those in `templates/routing-informs/part1.sh`
    - Updated "Press return" messages to be consistent with their functions

### CODE REVIEW

**Review by nmisek — DISMISSED** [review]

File: templates/routing-informs/part1.sh [hunk 1]
```diff
@@ -9,9 +9,12 @@ if [ "${NEXTMV_API_KEY}" = "" ]; then
   exit 1
 fi
 
-printf "\nEnter your initials: "
+printf "\nEnter your initials (letters only): "
 read -r ID
 
+## Convert initials to lowercase (to match expected App ID formatting)
+ID=(echo $ID | tr '[:upper:]' '[:lower:]')
```
Comment (racquesta) [inline templates/routing-informs/part1.sh:16]: This is a nice touch for UX.

File: templates/routing-informs/part2.sh [hunk 2]
```diff
@@ -107,14 +107,15 @@ Now let's run an experiment comparing the results of our updated version to that
...
+    --app-id $APPID \\
+	--experiment-id "compare-with-service-times" \\
+    --instance-ids "prod,staging" \\
 			--input-set-id denver-sample \\
-      --name "Compare v1 prod to v2 staging" \\
-			--description "Comparison with service times"
-Press return to create the staging instance, with our updated version assigned.
+    --name "Compare v1 prod to v2 staging" \\
+    --description "Comparison with service times"
+Press return to create an experiment on the production and staging instances
```
Comment (racquesta) [inline templates/routing-informs/part2.sh]:
```suggestion
Press return to create an experiment comparing the production and staging instances
```

I'm not sure if the above is more clear or not but that is how I think about it.

File: templates/routing-informs/part2.sh [hunk 3]
```diff
@@ -125,3 +126,14 @@ nextmv experiment batch start \
 			--input-set-id denver-sample \
       --name "Compare v1 prod to v2 staging" \
 			--description "Comparison with service times"
+
+
+cat << EOF
+====== Cloud user interface   ======
+The Nextmv cloud website (https://cloud.nextmv.io) provides a GUI to check the
+status of experiments as well as more details on and visualizations of
```
Comment (racquesta) [inline templates/routing-informs/part2.sh]:
```suggestion
status of experiments as well as results and visualizations
```

File: templates/routing-informs/part2.sh [hunk 4]
```diff
@@ -125,3 +126,14 @@ nextmv experiment batch start \
...
+the results from completed experiments.
```
Comment (racquesta) [inline templates/routing-informs/part2.sh]:
```suggestion
of completed experiments.
```

**Review by racquesta — DISMISSED** [review]
I'm not exactly the best at docs either, but I tried. :) I would say nothing blocking here in m comments.

**Review by racquesta — DISMISSED** [review]

**Review by chriskeyser — APPROVED** [review]

**Pull request merged**